automerge/src/types/opids.rs

```diff
@@ -74,6 +94,38 @@ impl OpIds {
     pub(crate) fn contains(&self, op: &OpId) -> bool {
         self.0.contains(op)
     }
+
+    #[cfg(feature = "storage-v2")]
+    pub(crate) fn get(&self, idx: usize) -> Option<&OpId> {
+        self.0.get(idx)
+    }
+}
+
+#[cfg(feature = "storage-v2")]
+fn are_sorted_and_unique<
+    'a,
+    I: Iterator<Item = &'a OpId>,
+    F: FnMut(&OpId, &OpId) -> std::cmp::Ordering,
+>(
+    mut opids: I,
+    mut f: F,
+) -> bool {
+    use std::cmp::Ordering;
+    let mut last = match opids.next() {
+        Some(e) => e,
+        None => return true,
+    };
+
+    for next in opids {
+        if f(last, next) == Ordering::Greater {
+            return false;
+        }
+        if last == next {
```

Comment:
Could this be a `Ordering::Equal` check on the above if statement?

Reply:
Done!